workflows/tasks/bedtools.wdl

```diff
@@ -1,164 +1,169 @@
 version 1.0
-# Bedtools tools
 
+# Bedtools tools
```

Comment:
@a-frantz - Not sure if you've seen an example like this for `sprocket format`. This comment likely should have been "promoted" to a preamble comment. I'm not sure if we can cleanly write a rule for that. Should all detached comments at the beginning of a WDL file become preamble comments?

Reply:
I think that would be tough to get right, and is more likely to annoy people than be helpful.